provisioning/playbooks/roles/transactor/tasks/main.yml

```diff
@@ -1,7 +1,3 @@
-- name: update system
```

Comment:
I think we should probably have this in a role? We need it in all the proviosioning scripts.